Using TensorFlow backend.
WARNING:tensorflow:From /cm/shared/apps/python/2.7.14-1804/lib/python2.7/site-packages/tensorflow/contrib/learn/python/learn/datasets/base.py:198: retry (from tensorflow.contrib.learn.python.learn.datasets.base) is deprecated and will be removed in a future version.
Instructions for updating:
Use the retry module or similar alternatives.
Traceback (most recent call last):
  File "train.py", line 374, in <module>
    label_suffix=label_suffix, ignore_label=ignore_label, label_cval=label_cval)
  File "train.py", line 84, in train
    classes=classes,
  File "/home/shasvatmukes/project/segmentation/Shasvat/Keras-FCN/models.py", line 449, in FCN_Vgg16_8s
    input_shape=(image_size[0], image_size[1],3))
  File "/cm/shared/apps/python/2.7.14-1804/lib/python2.7/site-packages/keras/applications/vgg16.py", line 176, in VGG16
    model.load_weights(weights_path)
  File "/cm/shared/apps/python/2.7.14-1804/lib/python2.7/site-packages/keras/engine/topology.py", line 2646, in load_weights
    raise ImportError('`load_weights` requires h5py.')
ImportError: `load_weights` requires h5py.
